src/Nethermind/Nethermind.TxPool/AcceptTxResult.cs

```diff
@@ -3,132 +3,155 @@
 
 using System;
 
-namespace Nethermind.TxPool
+namespace Nethermind.TxPool;
+
+public enum TxResultCode
+{
+    Accepted = 0,
+    AlreadyKnown,
+    FailedToResolveSender,
+    FeeTooLow,
+    FeeTooLowToCompete,
+    GasLimitExceeded,
+    InsufficientFunds,
+    Int256Overflow,
+    Invalid,
+    NonceGap,
+    OldNonce,
+    ReplacementNotAllowed,
+    SenderIsContract,
+    NonceTooFarInFuture,
+    PendingTxsOfConflictingType,
+    NotSupportedTxType,
+    MaxTxSizeExceeded,
+    NotCurrentNonceForDelegation,
+    DelegatorHasPendingTx,
+    PermissionDenied = 100,
+    Syncing = 503
+}
```

Comment:
So I think this needs to be extendable by plugins. Which is still is just pointing out why we used int